### PR TITLE
Set ecto application_name based on APP_NAME env var

### PR DESCRIPTION
**Pull request by zaremjan**

### CODE REVIEW

File: lib/surgex/repo_helpers/repo_helpers.ex [hunk 1]
```diff
@@ -83,4 +84,18 @@ defmodule Surgex.RepoHelpers do
       _ -> opts
     end
   end
+
+  @doc """
+  Sets application_name to the value of APP_NAME env var.
+  """
+  def set_application_name(opts) do
+    with env_value when is_binary(env_value) <- System.get_env("APP_NAME"),
+         application_name <- String.slice(env_value, 0, 63),
```
Comment (saner) [inline lib/surgex/repo_helpers/repo_helpers.ex:93]: Why do we limit to 63 characters?

File: test/surgex/repo_helpers/repo_helpers_test.exs [hunk 2]
```diff
@@ -78,4 +78,56 @@ defmodule Surgex.RepoHelpersTest do
...
+
+    assert opts[:parameters][:test_key] == "test_value"
+    assert opts[:parameters][:application_name] == "surgex"
+  end
+
+  test "set_application_name env var present, no existing params" do
+    System.put_env("APP_NAME", "app-name-from-env")
+    opts = RepoHelpers.set_application_name([])
+    assert opts[:parameters][:application_name] == "app-name-from-env"
+    System.delete_env("APP_NAME")
```
Comment (saner) [inline test/surgex/repo_helpers/repo_helpers_test.exs:104]: Shouldn't this be in the `after` clause?

Comment (zaremjan) [reply test/surgex/repo_helpers/repo_helpers_test.exs]: Is there an `after` clause in ExUnit? 
I've found `on_exit`, and I've pushed the commit that uses it.

File: lib/surgex/repo_helpers/repo_helpers.ex (hunk 1 above)
Comment (zaremjan) [reply lib/surgex/repo_helpers/repo_helpers.ex:93]: Quoting postgres docs:
>The application_name can be any string of less than NAMEDATALEN characters (64 characters in a standard build). 

https://www.postgresql.org/docs/13/runtime-config-logging.html#GUC-APPLICATION-NAME

**Review by saner — APPROVED** [review]
💯 looks good

File: test/surgex/repo_helpers/repo_helpers_test.exs (hunk 2 above)
Comment (zaremjan) [reply test/surgex/repo_helpers/repo_helpers_test.exs:105]: It turns out `after` is a elixir feature: https://elixir-lang.org/getting-started/try-catch-and-rescue.html#after

**Pull request merged**